package.json

```diff
@@ -15,7 +15,8 @@
     "preview": "vite preview",
     "test:unit": "vitest --run",
     "test:e2e": "playwright test",
-    "prepare": "husky"
+    "prepare": "husky",
+    "update-ds": "npm update @jod/design-system"
```

Comment:
This is already in the readme file.

Reply:
I know, I was just thinking that this is more convenient. Should I remove it or update the readme to use this command?

Reply:
Command removed